internal/controller/controllers/common.go

```diff
@@ -423,24 +447,26 @@ func isBaremetalPlatform(ctx context.Context, client client.Client, cd *hivev1.C
 	if err != nil {
 		return false, errors.Wrapf(err, "failed to get AgentClusterInstall %s platform", clusterInstall.Name)
 	}
+	if platform == nil {
+		return false, nil
+	}
 	return *platform.Type == models.PlatformTypeBaremetal, nil
 }
 
 // We get first agent's cluster deployment and then we query if it belongs to none platform cluster
```

Comment:
I don't think this comment is valid any more

Reply:
Good catch, modified it, thanks!